packages/waku/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "declaration": true,
+    "emitDeclarationOnly": true,
+    "rootDir": "./src",
+    "outDir": "./dist",
+    "types": ["react/experimental"],
```

Comment:
I prefer inline declaration instead of having it here.
Also, we shouldn't use `experimental`.

Reply:
https://github.com/dai-shi/waku/pull/332 I changed my mind for examples/templates.